tests/integration/pilot/main_test.go

```diff
@@ -44,3 +54,20 @@ func TestMain(m *testing.M) {
 		}).
 		Run()
 }
+
+func echoConfig(ns namespace.Instance, name string) echo.Config {
+	return echo.Config{
+		Service:   name,
+		Namespace: ns,
+		Ports: []echo.Port{
+			{
+				Name:     "http",
+				Protocol: protocol.HTTP,
+				// We use a port > 1024 to not require root
+				InstancePort: 8090,
+			},
+		},
+		Subsets: []echo.SubsetConfig{{}},
+		Pilot:   p,
+	}
+}
```

Comment:
Move the echoConfig to the main file since this function seems to be used across the suite.